projects/hslayers/src/common/layers/public-api.ts

```diff
@@ -1,3 +1,5 @@
 export * from './hs.source.SPOI';
 export * from './hs.source.SparqlJson';
 export * from './hs.source.WfsSource';
+export * from './interpolated-layer.class';
```

Comment:
Don't name the plain class a ".class". Everything is a class. Let's go with just "interpolated-layer.ts".

Reply:
Sorry about that!